### DISCUSSION TITLE
Contribution Overview From Past 2 Weeks not Showing Up

### DISCUSSION THREAD

**Discussion by MateuszKrolik** [community/community#87710] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, my Contribution Overview From Past 2 Weeks is not Showing Up, even though my local git mail that i push from to my remote repo matches my main github mail. I also enabled all possible visibility options on my profile to double check. Could please somebody help me locate the source of the issue? Thanks in advance.

**Comment by moheedinamdar** [comment]
Ensure that the email address associated with your local Git commits exactly matches the email address associated with your GitHub account. You can check your Git configuration using
`git config user.email`
If it doesn't match, you can set it using:
`git config --global user.email "your.email@example.com"`
AND
Confirm that the repositories where you are making contributions are either public or that you have the necessary access rights to view private repositories. Contributions in private repositories may not be visible to the public.
ALSO
If you're using GitHub Pages, activities related to the GitHub Pages repository may not be considered in the contribution graph. The graph typically considers activities in repositories other than GitHub Pages.
Hope this helps,
Thanks.
